modules/nf-core/sratools/fasterqdump/main.nf

```diff
@@ -23,7 +23,10 @@ process SRATOOLS_FASTERQDUMP {
     def args = task.ext.args ?: ''
     def args2 = task.ext.args2 ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def outfile = meta.single_end ? "${prefix}.fastq" : prefix
+    def outfile = "${prefix}.fastq"
```

Comment:
Why does this no longer distinguish between single-end and paired-end files? In the stub below you have
```groovy
def outfile = meta.single_end ? "${prefix}.fastq" : "${prefix}_1.fastq ${prefix}
```

I would think that `--outfile ${prefix}.fastq` for paired-end data causes confusion with filenames?

Reply:
fasterq-dump has two modes for specifying outfile prefix. The one which takes 'something' as prefix is currently broken due to '.' in the path bug. Please see: https://github.com/ncbi/sra-tools/issues/865

If we use 'something.fastq', luckily, the '.' in the path bug can be avoided. But this mode creates three files for some paired end SRAs with '.fastq' extension which are emitted by the process. To exclude the third file, `mv prefix.fastq prefix || echo '...'` is used.